templates/app/helpers/spree/product_options_helper.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Spree
+  module ProductOptionsHelper
+    def product_variants_with_options
+      @product.variants_and_option_values_for(current_pricing_options)
```

Comment:
What about passing `product` as an argument to this method? It will be consistent with the other method above and I think it's better because we could use it in contexts where we don't have the instance variable (eg. product listing)

Reply:
Sounds good. I'll update the methods to receive `product` as an argument. TODO.